src/decision_tree.cpp

```diff
@@ -63,15 +63,15 @@ DecisionTree::DecisionTree(
     this->max_leaves_ = max_leaves;
     this->min_obs_ = min_obs;
     this->max_prop_ = max_prop;
-    this->seed_ = seed;
+    this->meta_seed_ = seed;
```

Comment:
@gpestre shouldn't this still be `this->seed`? The meta-seed is set outside the tree, while the tree gets its seed from the master `SeedGenerator`?

Reply:
My understanding is the meta_seed_ seeds the SeedGenerator. In decision_tree.cpp Line 74, we create this->seed_gen as a seeded SeedGenerator, then produce data_seed and tree_seed in random_forest.cpp lines 120-121 from this. So I think the seed we give the DecisionTree becomes the meta_seed for SeedGenerator, which then produces what we want.

Reply:
Noting that this->seed_ was just passed into SeedGenerator, thus becoming meta_seed_ so I just standardized that.

Reply:
I think I intended it as @wfseaton described, but either way works.
Form the DT's perspective, it's a metaseed; but the RF has it's own metaseed (which is sort of the metametaseed), but that's waaaay too meta